examples/asr/emformer_rnnt/global_stats.py

```diff
@@ -42,7 +42,7 @@ def generate_statistics(samples):
 
     for idx, sample in enumerate(samples):
         mel_spec = spectrogram_transform(sample[0].squeeze()).transpose(1, 0)
-        scaled_mel_spec = piecewise_linear_log(mel_spec * GAIN)
+        scaled_mel_spec = piecewise_linear_log(mel_spec)
```

Comment:
If we remove the GAIN computation, does it mean the `global_stats.json` needs to be updated as well?

Reply:
the scaling by `GAIN` happens within `piecewise_linear_log`, so no

Reply:
Got it.